### DISCUSSION TITLE
Feature Request: Use rotten tomatoes to chase away the wandering trader

### DISCUSSION THREAD

**Discussion by joernott** [vectorwing/FarmersDelight#947] [Ideas]
### Minecraft version

1.20.1

### Farmer's Delight version

1.2.4

### Forge version

any supported

### Description

If there is one visitor who is more annoying than pillagers, it's the wandering trader. It would be a cool feature, if throwing a rotten tomato at a wandering trader causes him to run away (despawn)

### Steps to reproduce

Example:
1. Wandering trader groans in your ear and stands directly on the block you want to build on.
2. You throw a rotten tomato at him
3. He vanished into thin air, taking his llamas with him.

### Mod list

Farmers Delight

### Minimal instance

- [ ] I have tested this on a minimal instance

### Performance and shader mods

- [ ] I am using performance or shader mods

**Comment by vectorwing** [comment]
Heh, not a bad idea! 😄

I once had some concepts for a mod centered around the Wandering Trader, to fix some annoyances while adding a few new features to them. Not sure how easy it would be to implement something well-animated with the Rotten Tomato, though it could work.

**Reply by joernott** [reply]
I would just ~~steal~~use the snowball animation/logic as a starting point. However, I have not delved into the depth of how these things are coded, so this might not be feasible.
